app/screens/BaseScreen.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+import AddIcon from '@material-ui/icons/Add';
+
+import Button from '../components/Button/Button';
+import Paper from '../components/Paper/Paper';
+import Typography from '../components/Typography/Typography';
+
+const StyledScreenTitle = styled.div`
+  margin: 0 25px 25px 25px;
+`;
+const StyledScreenContainer = styled(Paper)`
+  padding: 25px;
+`;
+
+const StyledScreenContent = styled.div`
+  margin: 0 25px;
+`;
+
+const StyledFab = styled(Button)`
+  &&& {
+    margin: 0;
+    top: auto;
+    right: 20px;
+    bottom: 20px;
+    left: auto;
+    position: fixed;
+  }
+`;
+
+const StyledActionContainer = styled.div`
+  padding: 2px;
+`;
+
+BaseScreen.propTypes = {
+  actionHandler: PropTypes.func,
+  buttonIcon: PropTypes.element,
+  children: PropTypes.node.isRequired,
+  title: PropTypes.string.isRequired
+};
+
+BaseScreen.defaultProps = {
+  actionHandler: () => {},
+  buttonIcon: <AddIcon />
+};
+
+export function BaseScreen(props) {
+  const { actionHandler, buttonIcon, children, title } = props;
```

Comment:
Can we have a convention of naming the `handler props` with `on` prefix (onAction)? For event handlers, inside the components, we'll just go with the `handle` prefix just like what you did above.